src/modules/core/components/MembersList/MembersListItem.tsx

```diff
@@ -127,7 +139,22 @@ const MembersListItem = <U extends AnyUser = AnyUser>(props: Props<U>) => {
             </span>
           )}
           <div className={styles.address}>
-            <CopyableAddress>{walletAddress}</CopyableAddress>
+            <InvisibleCopyableAddress address={walletAddress}>
+              <MaskedAddress address={walletAddress} />
+            </InvisibleCopyableAddress>
+            {isWhiteListed && (
+              <IconTooltip
+                icon="check-mark"
+                tooltipText={MSG.whitelistedTooltip}
+                iconSize="13px"
```

Comment:
I am not a big fan of passing `13px` like this. It's not about your PR, it's coming form `IconTooltip`, I think it would be good to create an issue to add `appearance` prop with iconSize key and have several standard options there - similar to what is done in other core components...
@arrenv @ArmandoGraterol what do you think?

Reply:
@chinins I agree, good thinking!

Reply:
Right. On a related note, I also want to point out that the Tooltip/ Popover component we are using is a bit hacky. For example, you cannot right click and Inspect Element on a Tooltip-ed component - try the colony address for instance.